### ISSUE TITLE
BetterPushback not working with Sparky 747 XP11

### ISSUE THREAD

**Issue by SkaperrrAviation** [skiselkov/BetterPushbackC#453]
I try to use better pushback with the sparky 747 and the pushback cart is somewhere so far you cant even see the thing when you zoom out. Issue occurs one time after another after multiple sim restarts. Also once i enter the menu, it freezes my movement so i cant use my arrow keys to move anywhere in any view, NEEDS TO BE FIXED ASAP.
![image](https://user-images.githubusercontent.com/121579994/209867616-93d79588-f81d-4940-aca9-bcc0449ca5e1.png)
![image](https://user-images.githubusercontent.com/121579994/209867660-36766579-8100-4086-86d4-665b21075bcf.png)

**Comment by tukan68** [comment]
Pure curiosity: which airport is this?

**Comment by SkaperrrAviation** [comment]
> Pure curiosity: which airport is this? Found it: Boston! Nice.
Yes it is boston, this is happening because my monitors but im not sure how to re order the monitors layout in xplane because when i do what everyone says it puts it on my secondary monitor
